backend/gke/deploy.sh

```diff
@@ -31,7 +31,10 @@ node_count     = "$GKE_NODE_COUNT"
 vm_type        = "UBUNTU"
 gke_sa_key     = "$GKE_CRED_JSON"
 gcp_dns_sa_key = "$GKE_CRED_JSON"
-cluster_labels = {key = "$GKE_CLUSTER_NAME"}
+cluster_labels = {
+    catapult-clustername = "$GKE_CLUSTER_NAME",
```

Comment:
whats going on here? 
https://github.com/SUSE/cap-terraform/blob/master/gke/terraform.tfvars.template#L8 : states just key = value. 
may be a comment or something

Reply:
yeah, the template file just has placeholders. The proposed labels here look fine to me.

Reply:
`key = value` is the general syntax. Importantly, `key` can be any string.
In the old code the key was named `key`, which did not say much.
Now the key is called the more descriptive `catapult-clustername`.
This goes into the cluster label section of the AWS console.
IIRC this also allows search by key and/value in the console.
Makes finding the clusters and cleaning up after trouble easier.

Reply:
ok. But when CI will use this; `owner = "$(whoami)".` it will provide a value `root`. Which won't make much sense. 
It was better to keep it `{key = "$GKE_CLUSTER_NAME"}`, as that would allow local deployment and CI to use relevant label.
For local deployment if someone don't want to provide a relevant label, then it anyways defaults to` whoami` : `GKE_CLUSTER_NAME="${GKE_CLUSTER_NAME:-$(whoami)-cap}"`

Reply:
I took the `owner` label idea from Satadru's clusters on GKE, there were more visible than the cluster names. Also, Since the cluster name can be changed, I find value in having a label that is just owner and cannot be changed. Simplifies searching for your clusters.